### ISSUE TITLE
Update Strusoft.Interop.Struxml.Data to 22 version

### ISSUE THREAD

**Issue by andosca** [strusoft/femdesign-api#725]

**Comment by Marco-Pellegrino** [comment]
let's also wait the milestone as we will get (hopefully) an updated version that will deserialise moving load correctly

**Comment by andosca** [comment]
yes this is not urgent in any way. but we should do it at some point :) there is an issue with dimtext_type as well.

**Comment by Marco-Pellegrino** [comment]
have you open a case on Fogbugz?
Schmidt Dávid seems to be the developer in charge

**Comment by andosca** [comment]
https://strusoft.fogbugz.com/f/cases/125100/Dimtext_type-position-has-incorrect-type

**Comment by Marco-Pellegrino** [comment]
#726
